tool/util.py

```diff
@@ -0,0 +1,79 @@
+# Lint as: python3
+
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""utility functions used to build the property-value tree structure """
```

Comment:
Since this is the main file, could you give the file a more descriptive name? pv_tree_generator or something like that?

Reply:
This will not be the main file. Currently working on adding the utility functions that will be used in the main file step by step. Updated the .sh file to avoid confusion.